src/RoboSaverVirtualModule.sol

```diff
@@ -82,34 +85,43 @@ contract RoboSaverVirtualModule {
                                       MODIFIERS
     //////////////////////////////////////////////////////////////////////////*/
 
-    /// @notice Checks whether a call is authorized to trigger top-up or exec queue txs
+    /// @notice Enforce that the function is called by the keeper only
     modifier onlyKeeper() {
         if (msg.sender != keeper) revert NotKeeper(msg.sender);
         _;
     }
 
     /*//////////////////////////////////////////////////////////////////////////
-                    EXTERNAL METHODS: TOP-UP AGENTS
+                                  EXTERNAL METHODS
     //////////////////////////////////////////////////////////////////////////*/
 
-    /// @dev Check condition and determine whether a task should be executed by Gelato.
+    /// @notice Check if there is a surplus or deficit of $EURe on the card
+    /// @return canExec Whether a transaction needs to be
+    /// @return execPayload The payload of the needed transaction
+    // @todo is this return structure the most efficient? why not just return action, card and amount?
```

Comment:
maybe these @todo now can be removed after our convo and one of them being tackled on the prev pr

Reply:
794f28a